### ISSUE TITLE
prefixize fails for defs styles using fill: url(#name)

### ISSUE THREAD

**Issue by andest01** [kisenka/svg-sprite-loader#27]
I am running `svg-sprite-loader@0.0.17`.

`prefixize.js` seeks to adjust names of styles that use `url(#myNameHere)` to the appropriate generated ID. However, this fails for one of our use cases that has a `linearGradient` in a SVG `def` attribute. 

Here is an example SVG image on codepen: http://codepen.io/troutr/pen/VaKMMR
Note the `style` named `.simpleClass` uses a fill to `#simpleGradient`.
```html
<defs>
    <style>
    .simpleClass {
        fill: url(#simpleGradient);  /*This should get updated to a new name...??*/
    }
    
    .company-fill {
        fill: #18a7e0;
    }
    </style>
    <linearGradient id="simpleGradient" x1="5709.17" y1="20.01" x2="5724.1" y2="34.94" gradientUnits="userSpaceOnUse">
        <stop offset="0" stop-color="#003952" />
        ...etc etc
        <stop offset="0.99" stop-color="#003952" />
    </linearGradient>
</defs>
```
Note the gradient fill on the bottom half of the SVG image.
![Result image with gradient](http://i.imgur.com/AFb3HKA.png)

Running this SVG through `prefixize: true` produces the following results:
```html
<defs>
    <style>
    .simpleClass {
        fill: url(#simpleGradient);  /* NOT UPGRADED! */
    }
    
    .company-fill {
        fill: #18a7e0;
    }
    </style>
    <linearGradient id="CompanyIcon_778e02c09f8a5ed0e12a9f654379ecbe_simpleGradient" x1="5709.17" y1="20.01" x2="5724.1" y2="34.94" gradientUnits="userSpaceOnUse">
        <stop offset="0" stop-color="#003952"></stop>
        ...etc etc
        <stop offset="0.99" stop-color="#003952"></stop>
    </linearGradient>
</defs>
```
This results in the following image:
![Result image without gradient](https://i.imgur.com/7SOiyC7.png)
I noticed that `fill: url(#simpleGradient);` is not updated to `fill: url(#CompanyIcon_778e02c09f8a5ed0e12a9f654379ecbe_simpleGradient);`.

**Comment by princed** [comment]
Yes, unfortunately we do not support css (even inline) yet.

**Comment by gazpachu** [comment]
I have the same problem

**Comment by kisenka** [comment]
Shipped with 0.0.31
